pxtblocks/fields/field_note.ts

```diff
@@ -525,6 +550,44 @@ export class FieldNote extends Blockly.FieldNumber implements FieldCustom {
         return FieldNote.keyHeight / 2;
     }
 
+    protected getNeighboringKeys(freq: number) {
+        let lowerBound;
+        let upperBound;
+
+        const maxNote = this.minNote_ + this.nKeys_ - 1;
+
+        for (let candidate = this.minNote_; candidate <= maxNote; ++candidate) {
+            if (this.getKeyFreq(candidate) + this.eps > freq) {
+                upperBound = candidate;
+                break;
+            }
+            lowerBound = candidate;
+        }
+
+        // Clamp key range at the top and bottom
+        if (!lowerBound) {
+            return { 
```

Comment:
Remove trailing space to fix linting error:
```suggestion
            return {
```